ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/BeaconState.java

```diff
@@ -145,7 +151,10 @@ public BeaconState(
       ArrayList<Bytes32> batched_block_roots,
       // PoW receipt root
       Bytes32 processed_pow_receipt_root,
-      ArrayList<CandidatePoWReceiptRootRecord> candidate_pow_receipt_roots) {
+      ArrayList<CandidatePoWReceiptRootRecord> candidate_pow_receipt_roots,
+      // Eth1Data
+      Eth1Data latest_eth1_data,
+      List<Eth1DataVote> eth1_data_votes) {
```

Comment:
Looking at this now makes me realize we should convert the rest of the ArrayLists in the constructor to Lists. I'll just open an issue for this.